### DISCUSSION TITLE
Multiplayer beatmap queue order

### DISCUSSION THREAD

**Discussion by psehr** [ppy/osu#20268] [Ideas] [answered]
It would be nice to have a customizable order for the beatmap queue in multiplayer, something around this example:

![image](https://user-images.githubusercontent.com/45183671/189663253-80090e46-d4f8-4533-86fa-c274b1afb34f.png)

**Comment by peppy** [comment, marked as answer]
Would definitely be a nice addition. Although it may be limited to certain queue modes, since it would add some considerable complexity to the automatic round-robin implementation.
